packages/tests/tests/peer_exchange.node.spec.ts

```diff
@@ -137,4 +139,68 @@ describe("Peer Exchange", () => {
       expect(waku.libp2p.peerStore.has(await nwaku2.getPeerId())).to.be.true;
     });
   });
+
+  describe("compliance test", async function () {
+    this.timeout(25_000);
+
+    let waku: LightNode;
+    let nwaku1: Nwaku;
+    let nwaku2: Nwaku;
+
+    tests({
+      async setup() {
+        nwaku1 = new Nwaku("1");
+        nwaku2 = new Nwaku("2");
+        await nwaku1.start({
+          discv5Discovery: true,
+          peerExchange: true,
+        });
+
+        const enr = (await nwaku1.info()).enrUri;
+
+        await nwaku2.start({
+          discv5Discovery: true,
+          peerExchange: true,
+          discv5BootstrapNode: enr,
+        });
+
+        waku = await createLightNode();
+
+        await waku.start();
+        const nwaku2Ma = await nwaku2.getMultiaddrWithId();
+
+        setTimeout(async () => {
```

Comment:
Why is it wrapped in setTimeout?

Reply:
`this.timeout` at the start of the `describe` block should be good enough, no?